File: dbfit-2.x/dbfit-fitnesse/src/main/java/dbfit/fixture/SetParameter.java
package dbfit.fixture;

import fit.Parse;

/** 
 * simple wrapper for fixture symbol value setting, 
 * which takes care of NULL keyword and symbol loading
 * 
 */
public class SetParameter extends fit.Fixture{
	public static void setParameter(String name, String value){
		if (value == null || "null".equals(value.toString().toLowerCase())) {
			dbfit.util.SymbolUtil.setSymbol(name, null);
		} else if (value != null && value.toString().startsWith("<<")) {
			String varname = value.toString().substring(2);
			if (!name.equals(varname)) {
				dbfit.util.SymbolUtil.setSymbol(name, dbfit.util.SymbolUtil.getSymbol(varname));
			}
		} else
			dbfit.util.SymbolUtil.setSymbol(name, value);
	}
	@Override
	public void doTable(Parse table) {
		if (args.length!=2) throw new UnsupportedOperationException("Set parameter requires two arguments");
		setParameter(args[0], args[1]);
	}
}

File: dbfit-dotnet/core/src/fixture/SetParameter.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace dbfit.fixture
{
    public class SetParameter : fit.Fixture
    {
        public static void SetParameterValue(String name, Object value)
        {
            if (value == null || "null".Equals(value.ToString().ToLower()))
            {
                fit.Fixture.Save(name, DBNull.Value);
            }
            else if (value != null && value.ToString().StartsWith("<<"))
            {
                string varname = value.ToString().Substring(2);
                if (!name.Equals(varname))
                {
                    fit.Fixture.Save(name, fit.Fixture.Recall(varname));
                }
            }
            else
                fit.Fixture.Save(name, value);
        }
        public override void DoTable(fit.Parse table)
        {
            SetParameterValue(Args[0], GetArgumentInput(1, typeof(object)));
        }
    }
}
